opencga-app/app/scripts/azure/arm/mongodb/azuredeploy.json

```diff
@@ -0,0 +1,341 @@
+                }
+            }
+        ]
+    },
+    "resources": [
+        {
+            "name": "[variables('nsgName')]",
+            "type": "Microsoft.Network/networkSecurityGroups",
+            "apiVersion": "2018-08-01",
+            "location": "[resourceGroup().location]",
+            "properties": {
+                "securityRules": [
+                    {
+                        "name": "Port_27017",
+                        "properties": {
+                            "protocol": "*",
+                            "sourcePortRange": "*",
+                            "destinationPortRange": "27017",
+                            "sourceAddressPrefix": "*",
+                            "destinationAddressPrefix": "*",
+                            "access": "Allow",
+                            "priority": 1020,
+                            "direction": "Inbound",
+                            "sourcePortRanges": [],
+                            "destinationPortRanges": [],
+                            "sourceAddressPrefixes": [],
+                            "destinationAddressPrefixes": []
+                        }
+                    },
+                    {
+                        "name": "default-allow-ssh",
+                        "properties": {
+                            "protocol": "TCP",
+                            "sourcePortRange": "*",
+                            "destinationPortRange": "22",
+                            "sourceAddressPrefix": "*",
+                            "destinationAddressPrefix": "*",
+                            "access": "Allow",
+                            "priority": 1000,
+                            "direction": "Inbound",
+                            "sourcePortRanges": [],
+                            "destinationPortRanges": [],
+                            "sourceAddressPrefixes": [],
+                            "destinationAddressPrefixes": []
+                        }
+                    },
+                    {
+                        "name": "Port_80",
+                        "properties": {
+                            "protocol": "*",
+                            "sourcePortRange": "*",
+                            "destinationPortRange": "80",
+                            "sourceAddressPrefix": "*",
+                            "destinationAddressPrefix": "*",
+                            "access": "Allow",
+                            "priority": 1010,
+                            "direction": "Inbound",
+                            "sourcePortRanges": [],
+                            "destinationPortRanges": [],
+                            "sourceAddressPrefixes": [],
+                            "destinationAddressPrefixes": []
+                        }
+                    }
+                ]
+            },
+            "tags": {}
+        },
+        {
```

Comment:
As per discussion this moring, think I agree with @lawrencegripper all vnet/nsg stuff would be better together as is solution specific rather than mongo template specific?

Reply:
Since only mongoDB needs NSG rules (afaik), I kept it in mongodb template. Another option would be to place the rules in the vnet template, as I did in an earlier commit: https://github.com/opencb/opencga/blob/bc266e88682b1b0fdd41e1a860d9febd2fb38240/opencga-app/app/scripts/azure/arm/vnet/azuredeploy.json 

Not sure what the best option is, but happy to change it back

Reply:
Removed NSGs for now in https://github.com/opencb/opencga/pull/975/commits/753dc108567e15b7f0e6d604ff3c2f99adda42c5